### PR TITLE
Harmony

### PR DESCRIPTION
**Pull request by mrmaxmysterious**

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Quite a poor presence, look on my suggested changes and think about redoing said sections

**Comment by Slowlife01** [comment]
Add support for audio (timestamp etc), and for sub-pages too

File: websites/H/HarmonyRadio/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "name": "Milan",
+      "id": "700817449414557736"
+    },
+    "service": "Harmony",
+    "description": {
+      "en": "Harmony | You bring the Harmony to music."
+    },
+    "url": [
+      "weareharmony.net"
+    ],
```
Comment (EncryptedDev) [inline websites/H/HarmonyRadio/dist/metadata.json:12]: Only one URL so this will do: 
```suggestion
    "url":
      "weareharmony.net"
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/HarmonyRadio/presence.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}), timeofbrowsing = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        
+    };
+    presenceData.startTimestamp = timeofbrowsing;
+    if (window.location.pathname.toLowerCase() === "/maintenance") {
+        presenceData.largeImageKey = "default";
+        presenceData.details = "Harmony - Maintenance";
```
Comment (EncryptedDev) [inline websites/H/HarmonyRadio/presence.ts:12]: Remove and add to the `presenceData` object

File: websites/H/HarmonyRadio/presence.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        presenceData.state = "We will be back soon!";
+    }
+    if (window.location.pathname.toLowerCase() === "/") {
+        presenceData.largeImageKey = "default";
```
Comment (EncryptedDev) [inline websites/H/HarmonyRadio/presence.ts:16]: Remove and add to the presenceData object

**Comment by Slowlife01** [comment]
No activity

**Pull request closed without merging**